swift/extractor/trap/TrapArena.h

```diff
@@ -0,0 +1,70 @@
+//
+// Created by redsun82 on 18.01.22.
```

Comment:
These are autogenerated by an IDE, right? Do we want to keep them?

Reply:
uh, right, forgot to remove that! yeah, it was CLion before I removed the template